src/test/lib/ApiTest.cpp

```diff
@@ -335,23 +335,26 @@ void QuicTestValidateConfiguration()
 #endif // QUIC_DISABLE_TICKET_KEY_TESTS
 }
 
-static
-_Function_class_(QUIC_LISTENER_CALLBACK)
-QUIC_STATUS
-QUIC_API
-DummyListenerCallback(
-    HQUIC,
-    void* Context,
-    QUIC_LISTENER_EVENT* Event
-    )
+namespace
 {
-    CxPlatEvent* StopCompleteEvent = (CxPlatEvent*)Context;
-    if (StopCompleteEvent &&
-        Event->Type == QUIC_LISTENER_EVENT_STOP_COMPLETE) {
-        StopCompleteEvent->Set();
-        return QUIC_STATUS_SUCCESS;
+    _Function_class_(QUIC_LISTENER_CALLBACK)
+    template<typename T>
```

Comment:
Why is this templated? It should always be the type that matches MsQuicListener now, right?

Reply:
I see what you're doing below.... I guess it's ok for now.